test/cassandra.yml

```diff
@@ -36,7 +36,7 @@ items:
       matchLabels:
         app: cassandra
     serviceName: cassandra
-    replicas: 3
+    replicas: 2
```

Comment:
I am not sure if 2 replicas is really a good idea. Can we switch rather to a single node cassandra cluster?

Reply:
My first commit to this PR was to change to a single replica, but that resulted in the cassandra_tests/TestCassandraSparkDependencies test failing and a message somewhere (I don't remember exactly, but can find it if necessary) about 2 cassandra instances being required for a quorum.  

Do I need to do something other than change the replica count to change to a single node cluster?

Reply:
The schema create job configuration will have to change probably.

e.g. mode test https://github.com/jaegertracing/jaeger/blob/b2a159a67b3d937eb29fe2a98f5b05708d34f56a/plugin/storage/cassandra/schema/create.sh#L36

Given that will require changing the example let's go with 2 replicas in tests.